versioned_docs/version-2.x/advanced/analytics/plugins.mdx

```diff
@@ -366,3 +366,24 @@ Feel free to browse
 contribute to them if you have any ideas! 💡
 
 :::
+
+## Known issue
```

Comment:
**issue:** several issues here IMO
1. this isn't an issue per se
2. datalayer is specific to GTM, so it shouldn't be mentioned for this part
3. it is more related to the signature of `settings()`

**suggestion:** rework this MR to highlight this in https://deploy-preview-681--heuristic-almeida-1a1f35.netlify.app/docs/2.x/advanced/analytics/getting-started#configure-analytics and maybe add a note to the new section from GTM's example

Reply:
I added a section in getting started to mention the workaround, removed the issue mention, and updated GTM's documentation to mention it.

Please let me know if the wording isn't right still.